frank-flow/src/frontend/src/app/shared/services/flow-structure.service.ts

```diff
@@ -459,10 +472,13 @@ export class FlowStructureService {
     | monaco.editor.IIdentifiedSingleEditOperation[]
     | void {
     for (const [nodeId, editAttributes] of this.editAttributeQueue.entries()) {
-      const node = this.currentFile.flowStructure?.nodes.find(
-        (node: FlowStructureNode) => node.uid === nodeId
-      );
-
+      let node: FlowStructureNode | undefined;
+      node =
+        nodeId === 'ConfigurationSettingsUpdate'
+          ? this.currentFile.flowStructure?.configuration
+          : this.currentFile.flowStructure?.nodes.find(
+              (node: FlowStructureNode) => node.uid === nodeId
+            );
```

Comment:
Configuration should be a node, so this can all be done in a generic fashion (or otherwise). 

It is fine for now, but should be taken in with the refactor. It makes the code a lot harder to read, just to add one little thing. 

I'll make an issue for this.

Reply:
Added seprate function for the adding/editing of flow settings and renamed the old attributes function to nodeAttribute